app.drey.Elastic.json

```diff
@@ -0,0 +1,133 @@
+{
+    "app-id" : "app.drey.Elastic",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.vala"
```

Comment:
Vala is in the GNOME SDK. Why do you need this?

Reply:
The language server is in that extension. But true, not needed for flathub, only upstream.